queries.sql

```diff
@@ -31,3 +31,14 @@ SELECT id,escapet_attempts, name FROM animals WHERE escapet_attempts = (SELECT M
 SELECT id,escapet_attempts, name FROM animals WHERE escapet_attempts = (SELECT MAX(escapet_attempts) FROM animals);
  SELECT MIN (weight_kg), MAX (weight_kg) FROM animals;
  SELECT AVG(escapet_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-1' AND '2000-12-31';
+ UPDATE animals SET species_id =2 WHERE name LIKE '%mon';
+UPDATE animals SET species_id = 1 WHERE species_id IS NULL;
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Sam Smith') WHERE name = 'Agumon';
+UPDATE animals SET owner_id= (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell') WHERE name IN ('Gabumon','Pikachu');UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob') WHERE name IN ('Devimon','Plantmon');
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond') WHERE name IN ('Blossom','Squirtle','Charmander');
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester') WHERE name IN ('Angemon','Boarmon');
```

Comment:
* Kindly move Everything that populates the tables with data to the data.sql file. as stated in the [project requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Remember%20these%20insertions%20and%20modifications%20happen%20in%20data.sql.)